client/src/components/network-diagram/item.tsx

```diff
@@ -0,0 +1,149 @@
+import { CollapsibleTrigger } from '@radix-ui/react-collapsible';
+import { ChevronDown } from 'lucide-react';
+
+import { cn } from '@/lib/classnames';
+
+import { Organization, Project } from '@/types/generated/strapi.schemas';
+
+import { Category } from '@/hooks/networks';
+
+import { SlidingLinkButton } from '../ui/sliding-link-button';
+
+import Document from '@/styles/icons/document.svg';
+
+type PathProps = {
+  heightIndex: number;
+  category: Category;
+  isGranchild: boolean | undefined;
+  isFirstOfType: boolean | undefined;
+};
+
+const Path = ({ heightIndex, category, isGranchild = false, isFirstOfType = false }: PathProps) => {
+  if (typeof heightIndex === 'undefined') return null;
+  const ITEM_HEIGHT = 90;
+  const PADDING = 40;
+  const additionalHeight = isFirstOfType ? 20 : 0;
+  const topHeight = heightIndex * ITEM_HEIGHT + additionalHeight;
+  const END_LINE_PADDING = 2;
+
+  const pathProps = {
+    stroke: 'black',
+    strokeWidth: category === 'coordinator' ? 3 : 1,
+    strokeDasharray: category === 'funder' ? '3' : '0',
+    fill: 'transparent',
+  };
+
+  return (
+    <>
+      <svg
+        height={`${PADDING + topHeight}px`}
+        width="25"
+        className={`absolute -left-6`}
+        style={{
+          top: `-${topHeight}px`,
+        }}
+      >
+        <path
+          d={`M0,${isGranchild ? topHeight - ITEM_HEIGHT : '0'} L0,${topHeight + 20}`}
+          {...pathProps}
+        />
```

Comment:
I think there's an issue with the scale or something else because the vertical part of the link is not the same width as the curved part:
![Between items, the vertical part of the link is thinner while the curve part looks as intended](https://github.com/Orcasa-Platform/orcasa/assets/6073968/861f45a2-30ba-4163-aa6e-d56b2f7b7fdb)